servant-named/servant-named-core/src/Servant/Symbols.hs

```diff
@@ -0,0 +1,143 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- |
+-- This module uses ideas from the Chapter 10 of the book @Thinking with Types@ by Sandy Maguire.
```

Comment:
shouldn't this be in a separate package on hackage? maybe there is one already?

Reply:
Indeed, there was a way to use the `first-class-families` package directly. Still, some useful expressions were missing there so they're grouped into a module.